### ISSUE TITLE
Online data dumping

### ISSUE THREAD

**Issue by smolendawid** [lanpa/tensorboard-pytorch#69]
I think there is a necessary function that is lacking here. Often during training you want to stop the program, change hyperparameters like learning rate and continue training. As far as I understand, it is not possible, because SummaryWriter needs .close() method to write log on disk. 

Or maybe it is possible to write to the same run after using .close(), so that we can just use close() after for example each epoch?

**Comment by smolendawid** [comment]
Ok so I can see flush parameter inside the objects.  Although it cannot be changed, basically everything works as I'd like.

Sorry for the mix-up.
